kythe/release/kubernetes/buildkite/deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+#
+# Copyright 2019 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Script to deploy the kythe-repo/buildkite kubernetes agent.
```

Comment:
Usage?  Can we just rerun this script to update the environment?

Reply:
I'm not sure what you mean "the environment"

Reply:
The docker container used as the builder.

Reply:
Yes, but see the additional comment if you want to update the k8s pod.